.phive/phars.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phive xmlns="https://phar.io/phive">
-  <phar name="composer-normalize" version="^2.13.3" installed="2.13.3" location="./.phive/composer-normalize" copy="true"/>
-  <phar name="phar-io/phive" version="^0.14.4" installed="0.14.4" location="./.phive/phive" copy="true"/>
-  <phar name="php-cs-fixer" version="^2.16.7" installed="2.16.7" location="./.phive/php-cs-fixer" copy="true"/>
-  <phar name="phpcs" version="^3.5.8" installed="3.5.8" location="./.phive/phpcs" copy="true"/>
-  <phar name="phpmd" version="^2.9.1" installed="2.9.1" location="./.phive/phpmd" copy="true"/>
-  <phar name="psalm" version="^4.7.0" installed="4.7.0" location="./.phive/psalm" copy="true"/>
+    <phar name="composer-normalize" version="^2.13.3" installed="2.13.3" location="./.phive/composer-normalize"
+          copy="true"/>
+    <phar name="phar-io/phive" version="^0.14.4" installed="0.14.4" location="./.phive/phive" copy="true"/>
+    <phar name="php-cs-fixer" version="^2.16.7" installed="2.16.7" location="./.phive/php-cs-fixer" copy="true"/>
+    <phar name="phpcs" version="^3.5.8" installed="3.5.8" location="./.phive/phpcs" copy="true"/>
+    <phar name="phpmd" version="^2.9.1" installed="2.9.1" location="./.phive/phpmd" copy="true"/>
+    <phar name="psalm" version="^4.7.0" installed="4.7.0" location="./.phive/psalm" copy="true"/>
```

Comment:
While we have 4 spaces indent for XML files in general, what happens when this file is regenerated by PHIVE?  Will it revert back to 2 spaces indent and need reformatting?